AmazonChimeSDK.podspec

```diff
@@ -1,14 +1,14 @@
 Pod::Spec.new do |s|
-  s.name             = 'AmazonChimeSDK-Bitcode'
+  s.name             = 'AmazonChimeSDK'
   s.version          = '0.24.1'
-  s.summary          = 'Amazon Chime SDK for iOS with Bitcode support.'
+  s.summary          = 'Amazon Chime SDK for iOS.'
   s.description      = 'An iOS client library for integrating multi-party communications powered by the Amazon Chime service.'
   s.homepage         = 'https://github.com/aws/amazon-chime-sdk-ios'
   s.license          = 'Apache License, Version 2.0'
   s.author           = { 'Amazon Web Services' => 'amazonwebservices' }
   s.source           = { :http => "https://amazon-chime-sdk-ios.s3.amazonaws.com/sdk/0.24.1/AmazonChimeSDK-0.24.1.tar.gz" }
-  s.ios.deployment_target = '11.0'
+  s.ios.deployment_target = '12.0'
```

Comment:
Did get any issue building against 11.0 with Xcode 15?

Reply:
These is a warning: "The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 11.0, but the range of supported deployment target versions is 12.0 to 17.2.99."

While it builds and seems to run the demo ok, there is no guarantee customers won't have issues. 11.0 is no longer officially supported by Apple with Xcode 15 and all apps submitted to the App Store Connect will be required to build with Xcode 15. I'd rather not take that risk.